src/controllers/user/trackgroups/routes/{id}/index.js

```diff
@@ -204,12 +204,16 @@ module.exports = function () {
       const where = {
         creatorId: ctx.profile.id,
         id: ctx.params.id
+        // creatorId: '49d2ac44-7f20-4a47-9cf5-3ea5d6ef78f6', // creator_id from database
```

Comment:
Can we clean this up?

Reply:
Sure. I'll take it out in the next iteration.